backend/src/common/enums/api/index.ts

```diff
@@ -1,2 +1 @@
-export * from './api-path.enum';
-export * from './users-api-path.enum';
+export * from 'healthcare-shared/common/enums/api';
```

Comment:
```suggestion
export { Something } from 'healthcare-shared/common/enums';
```
Export only that you need